### PR TITLE
Feature/4051/UI bugs

### PR DESCRIPTION
**Pull request by wendidy**
[dockstore/dockstore/issues/4051](https://github.com/dockstore/dockstore/issues/4051)
Screenshots:
1.
image
2.
image
3. 
image
4. When screen is medium sized
image

### CODE REVIEW

**Review by garyluu — CHANGES_REQUESTED** [review]
1. In my opinion, the version should:
  - avoid breaking a word that can fit on its own line
  - if the word is too long, only then should it break
  
Looks like on prod, it does the first point but not the 2nd. The versions table I believe does both. Your PR looks like it does only the 2nd point but not the first.

2. This is fine i think

3. Seems worse:

image

4. No comment

File: src/app/search/search-workflow-table/search-workflow-table.component.html [hunk 1]
```diff
@@ -33,16 +33,18 @@
...
+      >
+      <mat-cell fxShow fxHide.lt-sm data-cy="descriptorType" *matCellDef="let workflow" fxFlex="20%">
         <div>
           <span class="nonclick-badge">{{ workflow?.descriptorType | uppercase }}</span>
         </div>
       </mat-cell>
     </ng-container>
     <ng-container matColumnDef="projectLinks">
       <mat-header-cell fxShow fxHide.lt-md *matHeaderCellDef>Project Links</mat-header-cell>
-      <mat-cell fxShow fxHide.lt-md *matCellDef="let entry" class="align-languages">
```
Comment (garyluu) [inline src/app/search/search-workflow-table/search-workflow-table.component.html:45]: Is this the only use of the class? If so, remove the class.

**Review by garyluu — APPROVED** [review]
Adding Charles because I have no opinion on number 4

**Review by NatalieEO — APPROVED** [review]

**Review by coverbeck — APPROVED** [review]

File: src/app/search/search-workflow-table/search-workflow-table.component.html (hunk 1 above)
Comment (wendidy) [reply src/app/search/search-workflow-table/search-workflow-table.component.html:45]: @garyluu I need to use media query on this, fxLayout.lt doesn't work in this case for some reason...

**Pull request merged**